packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -78,20 +80,20 @@ export default class SentryErrorLogger implements ErrorLogger {
         });
     }
 
-    private mapToSentryLevel(level: ErrorLevelType): Severity {
+    private mapToSentryLevel(level: ErrorLevelType): SeverityLevel {
         switch (level) {
-        case ErrorLevelType.Info:
-            return Severity.Info;
+            case ErrorLevelType.Info:
+                return 'info';
```

Comment:
This looks good, just out of curiosity should we have an enum to set all these values 
```
enum SeverityLevelEnum {
    INFO = 'info',
    ERROR = 'error',
    WARNING = 'warning',
    LOG = 'log',
    DEBUG = 'debug',
}
```

Reply:
What is the concern here not to use Sentry's [type defintion](https://github.com/getsentry/sentry-javascript/blob/9b7f43246e75934c5b689f04147be788d3589fc2/packages/types/src/severity.ts#L22)?

Reply:
We are still using Sentry type definition, since the method will still return a type `SeverityLevel`. This is just so we don't have strings in code and they are managed in one place.

Reply:
Updated. I didn't include `LOG` cause we only have four `ErrorLevelType`.
https://github.com/bigcommerce/checkout-js/blob/42ee27d679f81eaabc8b43771b08575525962d4a/packages/core/src/app/common/error/ErrorLogger.ts#L40